test/utils/skipIf.ts

```diff
@@ -15,13 +15,12 @@ export const describeSkipIf: (condition: boolean) => Mocha.PendingSuiteFunction
  * @param {boolean} condition - The condition to check.
  * @returns {Function} The test function.
  */
-export const testSkipIf: (condition: boolean) => Mocha.PendingTestFunction =
-  (it as any).skipIf ??
-  function testSkipIf(condition: boolean) {
-    return condition ? it.skip : it;
-  };
+export const testSkipIf: (condition: boolean) => Mocha.PendingTestFunction = (
+  condition: boolean,
+) => (condition ? it.skip : it);
 
 export const isJSDOM = /jsdom/.test(window.navigator.userAgent);
 export const isOSX = /macintosh/i.test(window.navigator.userAgent);
 export const hasTouchSupport =
   typeof window.Touch !== 'undefined' && typeof window.TouchEvent !== 'undefined';
+export const isVitest = process.env.VITEST === 'true';
```

Comment:
This doesn't seem to be used, not sure if it'll be used in a future PR, so just flagging it.

Reply:
Yeah, also in favor of removing it if it's not used. We should favor feature flagging over environment flags.